styles/style.css

```diff
@@ -1,9 +1,114 @@
 * {
   margin: 0;
   padding: 0;
-  box-sizing: 0;
+  box-sizing: border-box;
 }
 
+/*=====testimonial-section====*/
+.testimonials-section {
+  max-width: 1917px;
+  position: relative;
+  overflow: hidden;
+}
+.testimonials-section-mast {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding-block: 100px;
+}
+.testimonials-section-title-comman {
+  display: flex;
+  gap: 10px;
+  position: absolute;
+  top: 20px;
+  left: -20px;
+}
+.testimonials-section-imge {
+  display: block;
+  position: relative;
+}
+.testimonials-section-imge-man {
+  border-radius: 25px;
+}
+.testimonials-section-imge-circle {
+  position: absolute;
+  z-index: 1;
+  top: -59px;
+  right: -73px;
+  border-radius: 100%;
+  background: rgba(255, 255, 255, 0.25);
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(8px);
+  -webkit-backdrop-filter: blur(8px);
+  border: 80px solid rgba(255, 255, 255, 0.09);
+}
+.testimonials-section-imge-asset {
+  position: absolute;
+  top: 250px;
+  left: -90px;
+  background: rgba(255, 255, 255, 0.25);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(8px);
+  -webkit-backdrop-filter: blur(8px);
+  border: 1px solid rgba(255, 255, 255, 0.09);
+}
+.testimonials-section-title {
+  position: relative;
+  left: 50px;
+}
+.testimonials-section-subtitle {
+  font-family: "Syne, sans-serif";
+  font-size: 35px;
+  text-align: left;
+  position: relative;
+  margin-top: 50px;
+  max-width: 30ch;
+}
+.testimonials-section-title-comman-one {
+  opacity: 0.05px;
+}
+.testimonials-section-title-comman-two {
+  opacity: 0.05px;
+}
```

Comment:
İki farklı class aynı özellikleri barındırıyor, html tarafında ikisine de aynı class'ı ver. Css'te de tek bir kural olsun ikisi için.